2019-11-19 16:28:57,773 file.manager INFO 1.2.0
2019-11-19 16:28:57,773 file.manager INFO Copyright (C) 2003-2010
2019-11-19 16:28:57,773 file.manager INFO   Noriaki Ando
2019-11-19 16:28:57,773 file.manager INFO   Intelligent Systems Research Institute, AIST
2019-11-19 16:28:57,773 file.manager INFO Manager starting.
2019-11-19 16:28:57,775 file.manager INFO Starting local logging.
2019-11-19 16:28:57,779 file.manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-11-19 16:28:59,815 file.ManagerServant INFO Master manager not found
2019-11-19 16:28:59,821 file.LocalServiceAdmin INFO All the local services are enabled.
2019-11-19 16:28:59,823 file.LocalServiceAdmin INFO All the local services are enabled.
2019-11-19 16:28:59,844 file.manager INFO Component type conf file: Human_Tracking_py.conf loaded.
2019-11-19 16:28:59,855 file.ec_base INFO setRate(1000000.000000) done
2019-11-19 16:28:59,857 file.ec_base INFO setRate(10.000000) done
2019-11-19 16:28:59,861 file.rtobject INFO 1 execution context was created.
2019-11-19 16:28:59,875 file.rtobject INFO Initial active configuration set is default.
2019-11-19 16:29:05,770 file.SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-11-19 16:29:05,772 file.SdoServiceAdmin ERROR type  already exists.
2019-11-19 16:29:11,921 file.ManagerServant INFO Master manager not found
2019-11-19 16:29:24,061 file.ManagerServant INFO Master manager not found
2019-11-19 16:29:36,197 file.ManagerServant INFO Master manager not found
2019-11-19 16:29:40,539 file.rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1571, in on_execute
    ret = self.onExecute(ec_id)
  File "c:\workspace\Human_Tracking_py\Human_Tracking_py.py", line 200, in onExecute
    return RTC.RTC_OK
AttributeError: 'NoneType' object has no attribute 'RTC_OK'

2019-11-19 16:29:40,638 file.rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1370, in on_deactivated
    ret = self.onDeactivated(ec_id)
  File "c:\workspace\Human_Tracking_py\Human_Tracking_py.py", line 162, in onDeactivated
    return RTC.RTC_OK
AttributeError: 'NoneType' object has no attribute 'RTC_OK'

2019-11-19 16:29:48,309 file.ManagerServant INFO Master manager not found
